VRCFaceTracking/OSC/OSCMessage.cs

```diff
@@ -109,17 +113,61 @@ public OscMessage(byte[] bytes)
-
                     Value = Encoding.ASCII.GetString(stringBytes.ToArray());
                     break;
-                case 70:
+                case 98: // OSC Type tag: 'b' ; OSC-blob
+                    Logger.Error("Unimplemented OSC-blob Type detected: " + type + " for name " + Address);
+                    break;
+
+                #endregion
+
+                #region Non Standard OSC-Type/s
+
+                case 104: // OSC Type tag: 'h' ; 64 Bit Big-Endian
+                    Logger.Error("Unimplemented OSC-blob Type detected: " + type + " for name " + Address);
+                    break;
+                case 116: // OSC Type tag: 't' ; OSC-timetag
+                    Logger.Error("Unimplemented OSC-timetag Type detected: " + type + " for name " + Address);
+                    break;
+                case 100: // OSC Type tag: 'd' ; 64 Bit Big-Endian
+                    Logger.Error("Unimplemented Double Type detected: " + type + " for name " + Address);
+                    break;
+                case 83: // OSC Type tag: 'S' ; Type represented in OSC-string
+                    Logger.Error("Unimplemented 'Type' OSC-string Type detected: " + type + " for name " + Address);
+                    break;
+                case 99: // OSC Type tag: 'c' ; 32 bit ASCII
+                    Logger.Error("Unimplemented ASCII Type detected: " + type + " for name " + Address);
+                    break;
+                case 114: // OSC Type tag: 'r' ; 32 bit RGBA color
+                    Logger.Error("Unimplemented RGBA color Type detected: " + type + " for name " + Address);
+                    break;
+                case 109: // OSC Type tag: 'm' ; 4 bit MIDI. Each byte as: port id, status, data1, data2
+                    Logger.Error("Unimplemented MIDI Type detected: " + type + " for name " + Address);
+                    break;
+                case 70: // OSC Type tag: 'T' ; Represents true, No extra data
                     Value = false;
                     break;
-                case 84:
+                case 84: // OSC Type tag: 'F' ; Represents false, No extra data
                     Value = true;
                     break;
+                case 78: // OSC Type tag: 'N' ; Represents NIL (zero), No extra data
+                    Value = 0;
+                    break;
+                case 73: // OSC Type tag: 'I' ; Represents Infinitum (endlessly infinite), No extra data. Capping to 1.
+                    Value = 1.0f;
+                    break;
+                case 91: // OSC Type tag: '[' ; Represents start of an array.
+                    Logger.Error("Unimplemented Array Type detected: " + type + " for name " + Address);
+                    break;
+                case 93: // OSC Type tag: ']' ; Represents end of an array.
+                    Logger.Error("Unimplemented Array Type detected: " + type + " for name " + Address);
+                    break;
+
+                #endregion
+
```

Comment:
https://user-images.githubusercontent.com/11042095/222858150-0530b601-1b66-40e1-885e-f201259b0f89.mp4


Just GOTO default I beg you 🤣

Reply:
Yeeeeeaaaahhhh should default the unimplemented ones. Got a little too ambitious with the changes my b 😥.

Reply:
Resolved in https://github.com/benaclejames/VRCFaceTracking/pull/109/commits/da0730058c64d099efb15ac120645fa0e60ff468